dotCMS/src/main/java/com/dotmarketing/db/DotCMSInitDb.java

```diff
@@ -54,7 +62,7 @@ public static void InitializeDb() throws DotDataException {
 	private static void buildDefaultData() throws DotDataException {
 		try {
 			HostAPI hostAPI = APILocator.getHostAPI();
-			PublicCompanyFactory.createDefaultCompany();
+			LocalTransaction.wrap(() -> PublicCompanyFactory.createDefaultCompany());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "PublicCompanyFactory"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)